src/rcapi/api/templates.py

```diff
@@ -20,7 +21,7 @@
 router = APIRouter()
 
 config, UPLOAD_DIR, NEXUS_DIR, TEMPLATE_DIR = initialize_dirs()
-DATE_FORMAT = "%a, %d %b %Y %H:%M:%S GMT" 
+DATE_FORMAT = "%a, %d %b %Y %H:%M:%S %z" 
```

Comment:
`datetime.strptime()` is weird. It produces timezone-aware object with `%z`, but does not with `%Z`, even if hardcoded values like `UTC` or `GMT` are parsed by it. The [technical notes](https://docs.python.org/3/library/datetime.html#technical-detail) are not particularly helpful, though they do mention how at some point `%z` began producing aware objects. In any case, this change does work, but it requires the strings to be "fixed" with `.replace("GMT", "+0000")`, as seen in the changes below.

Reply:
@vedina ^^^ FWIW